src/store/getters.js

```diff
@@ -8,4 +8,30 @@ export default
     {
         return state.location;
     },
+    geoloc (state)
```

Comment:
Seems like a lot of getters just for user location and shop location...
userLocation is redundant.